aptos-move/framework/aptos-stdlib/sources/cryptography/multi_key.move

```diff
@@ -0,0 +1,42 @@
+module aptos_std::multi_key {
```

Comment:
Can you add some general description on what this module does and what the expected flow is?

Reply:
dont know what you mean by general flow

Reply:
The comment currently doesn't explain anything. Add more details about what a MultiKey is, please (although it's already part of the general auth flow because we want localized context). Also remove the empty line between comment and module; I think it might break docs generation